src/payload.ts

```diff
@@ -31,27 +38,24 @@ export type Payload =
 export interface TokenTransferPayload {
   readonly type: StacksMessageType.Payload;
   readonly payloadType: PayloadType.TokenTransfer;
-  readonly recipientAddress: Address;
+  readonly recipient: PrincipalCV;
   readonly amount: BigNum;
   readonly memo: MemoString;
 }
 
 export function createTokenTransferPayload(
-  recipientAddress: string | Address,
+  recipient: PrincipalCV,
```

Comment:
I think for ease of use this should still take a string `recipient: PrincipalCV | string`